Feature/component logo

DeepaliAgrwl opened the pull request:
Please review the logo component changes and let me know your comments

DeepaliAgrwl committed:
Logo Component Changes

Initial component structure of logo component

DeepaliAgrwl committed:
logo style chnages

Committed logo style changes

DeepaliAgrwl committed:
logo component css text change

rakeshkumar-verma commented on ui.content/src/main/content/jcr_root/content/hclecomm/us/en/HomePage/.content.xml:
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
```
@DeepaliAgrwl  Why home page checkedin as  a part of logo component?

DeepaliAgrwl replied:
I created it for testing. But yes this should not be a part of testing. I have removed it from this branch and checked in the code as well

DeepaliAgrwl committed:
Home Page deleted

Home Page deleted as part of logo component

rakeshkumar-verma commented on ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml:
```diff
@@ -7,12 +7,23 @@
         <foundation jcr:primaryType="nt:unstructured">
             <components jcr:primaryType="nt:unstructured">
                 <responsivegrid jcr:primaryType="nt:unstructured">
+                    <policy_97222330617899
+                        jcr:lastModified="{Date}2020-04-21T14:02:13.092+05:30"
+                        jcr:lastModifiedBy="admin"
+                        jcr:primaryType="nt:unstructured"
```
@DeepaliAgrwl  Why this pull request have policy and template related files?

DeepaliAgrwl replied:
somehow the changes of template were merged on my local. Reverted back from this branch. Please have a look

DeepaliAgrwl committed:
reverted back template changes

reverted back template changes from this branch

rakeshkumar-verma commented on ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/clientlib/.content.xml:
```diff
@@ -0,0 +1,5 @@
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    allowProxy="{Boolean}true"
+    categories="[hclecomm.logo,hclecomm.structure]"/>
```
@DeepaliAgrwl  Keep only one category -hclecomm.logo and update other files accordingly

DeepaliAgrwl replied:
@rakeshkumar-verma removed one category. keeping one category as heclecomm.logo

DeepaliAgrwl replied:
also changed the respective base-clientlib for this

rakeshkumar-verma commented on ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/.content.xml:
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
@DeepaliAgrwl  Why this file is part of logo comp?

DeepaliAgrwl replied:
reverted back this file

DeepaliAgrwl committed:
Review Comments incorporated

1. keeping only one category of logoo component, removed ther one
2. reverted back /templates\.content.xml file

DeepaliAgrwl committed:
changed base clientob as well
